libraries/AP_HAL_Linux/SPIDevice.cpp

```diff
@@ -123,6 +123,12 @@ SPIDesc SPIDeviceManager::_device[] = {
     SPIDesc("aeroio", 1, 1, SPI_MODE_0, 8, SPI_CS_KERNEL,  10*MHZ, 10*MHZ),
     SPIDesc("bmi160", 3, 0, SPI_MODE_3, 8, SPI_CS_KERNEL, 1*MHZ, 10*MHZ),
 };
+#elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_EDGE
+SPIDesc SPIDeviceManager::_device[] = {
+    SPIDesc("mpu60x0",    0, 1, SPI_MODE_0, 8, SPI_CS_KERNEL,  1*MHZ, 20*MHZ),
+    SPIDesc("mpu60x0ext",    0, 2, SPI_MODE_0, 8, SPI_CS_KERNEL,  1*MHZ, 20*MHZ),
```

Comment:
Are you sure 20 is ok rather than 11 like the others? See aadc1643fca834487f38584e9268f2f947e7c2b7

Reply:
Thanks, Lucas. I have seen some temperature-related FIFO resets. Thank you for noticing this. I don't see a reason to maximize the frequency in this case, so I'll just do the same thing as others.

But I'd really like to hear a little more how did you glue this reset to the SPI speed.

Reply:
I think I noticed the resets right after rebasing and checking the IMU driver started monitoring the temperature. It was a long time ago, I don't remember exactly :(